test/functional/config.firefox.js

```diff
@@ -16,7 +16,9 @@ export default async function ({ readConfigFile }) {
       require.resolve('./apps/console'),
       require.resolve('./apps/dashboard/group4/dashboard_save'),
       require.resolve('./apps/dashboard_elements'),
-      require.resolve('./apps/discover'),
+      require.resolve('./apps/discover/classic'),
```

Comment:
```suggestion
      require.resolve('./apps/discover/classic'),
      require.resolve('./apps/discover/embeddable'),
```

We should also include the new group here, no?

Reply:
The embeddable suite doesn't actually have any `includeFirefox` tests so I didn't include it